packages/@o3r/localization/schematics/localization-base/index.ts

```diff
@@ -98,12 +100,12 @@ export function updateLocalization(options: { projectName?: string | null | unde
         locales: [
           'en-GB'
         ],
-        assets: [`${mainAssetsFolder}/locales`],
-        outputPath: `${devResourcesFolder}/localizations`
+        assets: [`${projectBasePath}/${mainAssetsFolder}/locales`],
```

Comment:
shouldn't we add a test somewhere to be sure we will not remove that fix at some point ? or understand if we should supports both cases

Reply:
We do not have e2e for an application outside of the mono-repo.
I can add a UT